### PR TITLE
Use @OSBotify's GPG key instead of @Botify

### PR DESCRIPTION
**Pull request by AndrewGable**
### Details

In my original PR https://github.com/Expensify/react-native-onyx/pull/137 I was using the @botify key and user, but instead we should be using the @OSBotify account.

### Related Issues

https://github.com/Expensify/Expensify/issues/211122

### Automated Tests

1. Merge PR
2. Verify PR creates a new npm version

### CODE REVIEW

**Comment by AndrewGable** [comment]
On hold until we do this: https://github.com/Expensify/Expensify/issues/211122

**Comment by AndrewGable** [comment]
Off hold

**Review by roryabraham — CHANGES_REQUESTED** [review]
GH won't let me comment on line 13 but we need:

```
if: ${{ github.actor != 'OSBotify' }}
```

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -23,28 +23,28 @@ jobs:
...
               run: |
                 cd .github
-                gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output Botify-private-key.asc Botify-private-key.asc.gpg
-                gpg --import --batch Botify-private-key.asc
+                gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output OSBotify-private-key.asc OSBotify-private-key.asc.gpg
+                gpg --import OSBotify-private-key.asc
               env:
                 LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
 
             - name: Set up git for Botify
```
Comment (roryabraham) [inline .github/workflows/publish.yml:34]: NAB

```suggestion
            - name: Set up git for OSBotify
```

**Review by roryabraham — DISMISSED** [review]

**Review by roryabraham — APPROVED** [review]

**Pull request merged**